setup.py

```diff
@@ -58,7 +57,7 @@
         'rules',
         'uritemplate',
         # Production-only
-        'django-composed-configuration[prod]',
+        'django-composed-configuration[prod]>=0.16',
```

Comment:
See L67 too where `django-composed-configuration[dev]` is listed. I don't think setup tools will grab the version from the default section in `dev` mode